portfolio/src/main/webapp/index.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<DOCTYPE html>
```

Comment:
please revert this line - this should be <!DOCTYPE html> (it's a special declaration for html5)